exporter/instanaexporter/README.md

```diff
@@ -57,9 +58,9 @@ exporters:
   logging:
     loglevel: debug
   instana:
-    loglevel: debug
     endpoint: ${INSTANA_ENDPOINT_URL}
     agent_key: ${INSTANA_AGENT_KEY}
+    ca_file: someCA.pem # Optional. See the parameter table above for details.
```

Comment:
Optional. See the parameter table above for details. —> Optional. Certificate authority file for Instana backend connection.